pos_customer_required/models/pos_make_payment.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class PosMakePayment(models.TransientModel):
+    _inherit = "pos.make.payment"
+
+    def check(self):
+        # Load current order
+        order_obj = self.env["pos.order"]
+        order = order_obj.browse(self.env.context.get("active_id", False))
+
+        # Check if control is required
```

Comment:
```suggestion
        order = self.env["pos.order"].browse(self.env.context.get("active_id", False))
```

Nitpicking here but would be nice to remove the obvious comments and do a one-liner